src/root_files/app/index.html

```diff
@@ -49,14 +48,6 @@
                 var is_logged_in = client_info && client_info['token'];
                 window.location.href = (lang || 'en').toLowerCase() + '/' + (is_logged_in ? 'trading' : 'home') + '.html' + window.location.search;
             }
-            function isDelayedRedirect() {
-                if(isGtmApplicable() && !redirectTimeOut) {
-                    redirectTimeOut = setTimeout(redirect, 5000);
-                } else {
-                    clearTimeout(redirectTimeOut);
-                    redirect();
-                }
-            }
             function isGtmApplicable() {
                 return /(www|staging)\.binary\.com/i.test(window.location.hostname);
```

Comment:
How about `binary.me` and `desktop_app` handling?

Reply:
desktop_app has the same URL right? I'll just add the handler for binary.me